docs/blog/2025-03-19-using-angular-with-rspack.md

```diff
@@ -0,0 +1,166 @@
+
+## What about Rspack?
+
+[Rspack](https://rspack.dev) is a high performance JavaScript bundler written in Rust. It offers strong compatibility with the Webpack ecosystem, allowing for almost seamless replacement of webpack, and provides lightning fast build speeds.
+
+Because it supports the existing Webpack ecosystem, it provides an answer to teams that maintain Angular applications using Webpack and want to migrate to a faster build pipeline.
+
+However, it is crucial to understand that Rspack is not _completely_ compatible with the Webpack ecosystem with some slight nuances and low-level api differences that prevent certain plugins and loaders from working out of the box.
+One such example is the `AngularIvyPlugin` which is a Webpack plugin that is used to support Angular's Ivy compiler. Therefore, it was not possible to simply drop in Rpsack and expect it to work with Angular applications.
+Many people have tried to get Rspack working with Angular, but it has proven to be a challenge, with partial support and success being reported in the past but with a lot of limitations or actual performance degradations over the Webpack approach.
+
+Instead, a new approach was needed to support Rspack with Angular. A closer examination and replication of how Angular compiles and bundles the application was required.
+
+## Introducing Angular Rspack
+
+![Angular Rspack Logo](/blog/images/2025-03-19/nx-angular-rspack-logo-small.png)
+
+Angular Rspack started in September 2024 after I spent way too long investigating and researching how exactly Angular compiles and bundles for both their Webpack support and Esbuild support.
+
+Something that I kept coming back to was that any Rspack solution that relied too much on Angular's Webpack support had the chance of being dropped by the Angular team as they continue to build out incredible new features with Esbuild. Instead, I decided that replicating and utilizing the abstractions the Angular team provided for their Esbuild support had a much stronger chance of longevity.
+
+We are thrilled to announce that Angular Rspack is now being maintained by the Nx team and is available for use with Nx. This means that you can now use Rspack with Angular and Nx and enjoy the benefits of both.  
+The new package is called [@nx/angular-rspack](https://www.npmjs.com/package/@nx/angular-rspack) and it is available on npm while the repository has been moved to [nrwl/angular-rspack](https://github.com/nrwl/angular-rspack).
+
+We believe that a large portion of Angular applications still using Webpack will be able to migrate to Angular Rspack without issue.
+
+There are some limitations and missing features that are currently being worked on and on the roadmap to support. They have been listed at the bottom of this article.
+
+## Migrating from Angular Webpack to Angular Rspack
```

Comment:
I'd love another section before this that talks about the technical details under the hood. Like how Angular Rspack integrates into the Angular build pipeline. I'm very curiuos about how that works, like the actual low-level tech details and I bet many other Angular devs are too. 

That might also be the section that outlines some of the current limitations and improvements that are being worked on.

Then I'd shuffle the sections a little like in this order

- Using Angular Rspack
- Build and Serving your Application
- Running a configuration
- Benchmarks
- (Known Limitations and Missing Features) => I'd wire this into the "how it works" section
- Migrating from Angular Webpack to Angular Rspack